plugins/uf2/README.md

```diff
@@ -0,0 +1,43 @@
+# UF2 Devices
+
+## Introduction
+
+This plugin allows the user to update any supported UF2 Device by writing
+firmware onto a mass storage device.
+
+This is all specified here: <https://github.com/Microsoft/uf2>
+
+## Firmware Format
+
+The daemon will decompress the cabinet archive and extract a firmware blob in
+the UF2 file format.
+
+This plugin supports the following protocol ID:
+
+* com.microsoft.uf2
+
+## GUID Generation
+
+These devices use standard block DeviceInstanceId values, e.g.
+
+* `BLOCK\UUID_E478-FA50`
+* `BLOCK\LABEL_FWUPDATE`
```

Comment:
since you have UF2 ones now, how about drop these?

Reply:
Do you think we can rely on a valid USB VID/PID? I'm worried about a devboard having the same PID and a different UUID. Maybe we want `VID_1234&PID_5678&UUID_E478-FA50`` as an additional instance ID?

Reply:
Okay, I've dropped LABEL completely, and done the `VID_1234&PID_5678&UUID_E478-FA50` thing.

Reply:
the UUID thing I do feel like is so specific it only works for a single devboard.  Maybe it needs a description in README not to rely upon it for anything other than prototyping.

Reply:
So maybe we nuke the UUID thing completely and then re-add it if we find a board where the VID/PID is insufficient?